link/src/main/java/com/stripe/android/link/ui/inline/LinkInlineSignupView.kt

```diff
@@ -188,26 +188,29 @@ internal fun LinkInlineSignup(
                         AnimatedVisibility(
                             visible = signUpState == SignUpState.InputtingPhone
                         ) {
-                            Column(modifier = Modifier.fillMaxWidth()) {
-                                // TODO(brnunes-stripe): Migrate to phone number collection element
-                                PhoneCollectionSection(
-                                    phoneNumber = phoneNumber,
-                                    onPhoneNumberChanged = {
-                                        phoneNumber = it
-                                        if (phoneNumber.length == 10) {
-                                            onPhoneInput(phoneNumber)
-                                            keyboardController?.hide()
-                                        } else {
-                                            onPhoneInput(null)
+                            PaymentsTheme {
```

Comment:
Is this needed? Won't the call in Line 131 still be in scope here?

Reply:
Yup not needed. I removed it.